test/skimNtuple_HHbtag.cpp

```diff
@@ -4110,145 +4065,127 @@ int main (int argc, char** argv)
 		  //compute and store variations
 		  for (int isource = 0; isource < N_jecSources; isource++)
 			{
-			  // compute all shifted bjets
-			  tlv_firstBjet_raw_jetup[isource]   = tlv_firstBjet_raw ;
-			  tlv_firstBjet_raw_jetdown[isource] = tlv_firstBjet_raw ;
-			  tlv_secondBjet_raw_jetup[isource]  = tlv_secondBjet_raw;
-			  tlv_secondBjet_raw_jetdown[isource]= tlv_secondBjet_raw;
-
```

Comment:
I might be wrong but these lines were necessary to have meaningful values when we skim date

Reply:
What do you mean by "skim date"?
Regardless, those variables are defined [as follows](https://github.com/LLRCMS/KLUBAnalysis/blob/b54f30d953753820e63baa3d71e9ce50bf79a424/test/skimNtuple_HHbtag.cpp#L4022-L4025):

```c++
vector <TLorentzVector> tlv_firstBjet_jetup   (N_jecSources, tlv_firstBjet);
vector <TLorentzVector> tlv_firstBjet_jetdown (N_jecSources, tlv_firstBjet);
vector <TLorentzVector> tlv_secondBjet_jetup  (N_jecSources, tlv_secondBjet);
vector <TLorentzVector> tlv_secondBjet_jetdown(N_jecSources, tlv_secondBjet);
```

making the code above redundant (and slower).
Note that I've remove the "raw" substring from all the variables to avoid duplicates and confusion.

Reply:
I meant skim data.
These variables must be defined also for data.
And for some reason in the diff I did not see they were defined in a different place.
Looking at the file it seems fine.